rclc/QUALITY_DECLARATION.md

```diff
@@ -0,0 +1,200 @@
+
+## Version Policy [1]
+
+### Version Scheme [1.i]
+
+`rclc` uses `semver` according to the recommendation for ROS Core packages in the [ROS 2 Developer Guide](https://docs.ros.org/en/rolling/Contributing/Developer-Guide.html#versioning).
+
+### Version Stability [1.ii]
+
+`rclc` is at a stable version, i.e. `>= 1.0.0`.
+The current version can be found in its [package.xml](package.xml), and its change history can be found in its [CHANGELOG](CHANGELOG.rst).
+
+### Public API Declaration [1.iii]
+
+All symbols in the installed headers are considered part of the public API.
+
+All installed headers are in the [`include`](./include/rclc) directory of the package, headers in any other folders are not installed and considered private.
+
+### API Stability Policy [1.iv]
+
+`rclc` will not break public API within a released ROS distribution, i.e. no major releases once the ROS distribution is released.
+
+### ABI Stability Policy [1.v]
+
+`rclc` contains C code and therefore must be concerned with ABI stability, and will maintain ABI stability within a ROS distribution.
+
+### ABI and ABI Stability Within a Released ROS Distribution [1.vi]
+
+`rclc` will not break API nor ABI within a released ROS distribution, i.e. no major releases once the ROS distribution is released. Any ABI break in **rclc** will be done between minor versions and it should be clearly stated in the release notes.
+
+## Change Control Process [2]
+
+The stability of **rclc** is ensured through reviews, CI and tests.
+The change control process can be found in [CONTRIBUTING](../CONTRIBUTING.md).
+
+All changes to **rclc** occur through pull requests that are required to pass all CI tests.
+In case of failure, only maintainers can merge the pull request, and only when there is enough evidence that the failure is unrelated to the change.
+Additionally, all pull requests must have at least one positive review from another contributor that did not author the pull request.
+
+### Change Requests [2.i]
+
+All changes will occur through a pull request.
+
+### Contributor Origin [2.ii]
+
+This package uses [Developer Certificate of Origin (DCO)](https://developercertificate.org/) as its confirmation of contributor origin policy since version 1.0.0. More information can be found in [CONTRIBUTING](../CONTRIBUTING.md).
+
+### Peer Review Policy [2.iii]
+
+All pull requests will be peer-reviewed by at least one other contributor who did not author the pull request. Approval is required before merging.
+
+### Continuous Integration [2.iv]
+
+All pull requests must pass CI to be considered for merging, unless maintainers consider that there is enough evidence that the failure is unrelated to the changes.
+CI testing is automatically triggered by incoming pull requests.
+Current results can be seen [../README.md](../README.md).
+
+## Documentation [3]
+
+TODO check completeness
```

Comment:
```suggestion
```
Still a to-do or already solved?

Reply:
This was a reminder for me to check the list for completeness. Some items were missing. They are now included:
See commit https://github.com/ros2/rclc/pull/74/commits/1eb6bd89b4e09e30b874b3cf8772ed80e5ad6ced